osidb/filters.py

```diff
@@ -194,6 +201,19 @@ class Meta:
             + LTE_GTE_LOOKUP_EXPRS
             + DATE_LOOKUP_EXPRS,
             "affects__trackers__errata__advisory_name": ["exact"],
+            # Acknowledgment fields
+            "acknowledgments__uuid": ["exact"],
```

Comment:
I am unsure whether we need all these filter fields (e.g. `"acknowledgments__created_dt": ["exact"]` does not make sense to me). @JakubFrejlach Could you please look at it?

Reply:
Well, it is unlikely that someone will need to filter out exact datetime but no strong opinion to keep that or remove that.

Reply:
After some thought, the timestamps are unique to the given osidb instance and represent the time the osidb instance learned of the given acknowledgment, not the time when it was actually added into bugzilla. I'll try to reasonably trim the amount of filters in a separate commit. Thanks!

Reply:
Fixed in commit 627341b808e39ae2bd298779e83eabf3aab3ca9a

Reply:
> After some thought, the timestamps are unique to the given osidb instance and represent the time the osidb instance learned of the given acknowledgment, not the time when it was actually added into bugzilla. I'll try to reasonably trim the amount of filters in a separate commit. Thanks!

Not sure if I understand this correctly but the timestamps should be inherited from the flaw. Bugzilla is still the authoritative source of the data so OSIDB should just mirror it.

Reply:
Aha, thank you for the correction. `convertors.py` literally contains `created_dt=self.flaw_bug["creation_time"]` when setting `created_dt` of a `FlawAcknowledgment`.

In that light, it's IMO still right to remove `FlawAcknowledgment`'s filtering based on `created_dt`/`updated_dt` because `Flaw` can already be filtered based on `created_dt`/`updated_dt`.

Reply:
It is not necessary now but in the future it might be useful. I would leave it there. I am not sure if exact match is useful for the timestamps but generally I see the timestamps filtering handy.

Reply:
In that case, reverted commit 627341b808e39ae2bd298779e83eabf3aab3ca9a in commit d0354a6c6b5652baaee4919d2a446f020b056bc2 and the end result of this conversation is no change.